### ISSUE TITLE
LHN - Room chat remain bold when a different member sends a mention and deletes it.

### ISSUE THREAD

**Issue by vincdargento** [Expensify/App#53670]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.72-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Yes, reproducible on both
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** ibellicotest+71@gmail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Open the Expensify app.
2. Log in to an account with a room available but don´t open it.
3. From another device or environment, log in to a different account with access to the same room.
4. User B, mention one or more users that are not part of the room.
5. User A, verify that the chat preview in LHN is now bold.
6. User B, delete the mention.
7. User A, verify that chat preview.

## Expected Result:
Room chat preview, should not be displayed bold anymore, when the mentions sent by other member, are deleted.

## Actual Result:
Room chat remains bold in LHN, after a different member sends a mention of users that are not part of the room and deletes it.

## Workaround:
Unknown

## Platforms:
- [x] Android: Standalone
- [x] Android: HybridApp
- [x] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/08abd758-2f80-49df-928e-724bbe0b39ce

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by CortneyOfstad** [comment]
Thank you @NJ-2020!

**Comment by CortneyOfstad** [comment]
Still waiting on engineering to pick this up

**Comment by CortneyOfstad** [comment]
Still waiting on this to be picked up

**Comment by CortneyOfstad** [comment]
Still a hot pick!

**Comment by CortneyOfstad** [comment]
Just a heads up — I will be OOO starting this afternoon (December 20th) and will be returning January 6th. A handful of folks on the BZ team will be online for a few days in between the 25th and the 1st, but we'll be operating with a skeleton crew. If any action is needed from a BZ perspective, please post this issue in #expensify-open-source and someone on the team will jump in. Otherwise I'll review when I'm back in office. 

Thank you and Happy Holidays!

**Comment by CortneyOfstad** [comment]
This has been pretty stale so going to see if this can be retested 👍

**Comment by CortneyOfstad** [comment]
Still waiting for this to be tested!

**Comment by CortneyOfstad** [comment]
Bumped QA to have this retested [here](https://expensify.enterprise.slack.com/archives/C9YU7BX5M/p1739291282587339)

**Comment by kavimuru** [comment]
Reproducible

https://github.com/user-attachments/assets/d6ab34ee-571e-47c2-99c5-1320139ba2eb

**Comment by CortneyOfstad** [comment]
Refreshing the Hot Pick tag to see if that kick starts something

**Comment by CortneyOfstad** [comment]
Still a hot pick

**Comment by CortneyOfstad** [comment]
Still a hot pick

**Comment by JS00001** [comment]
Picking this up

**Comment by CortneyOfstad** [comment]
Thank you @JS00001!

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by JS00001** [comment]
Still reproducible on my end

With a ping:
<img width="976" alt="Image" src="https://github.com/user-attachments/assets/1fb43a89-3bcd-4a04-88b5-9490b9c73e58" />

When the ping is deleted:

<img width="864" alt="Image" src="https://github.com/user-attachments/assets/742af2a7-a3a8-4426-8da4-785bc71680df" />
